detox/src/utils/exec.js

```diff
@@ -43,7 +43,7 @@ async function execWithRetriesAndLogs(bin, options, statusLogs, retries = 10, in
     throw new DetoxRuntimeError(`command ${cmd} returned undefined`);
   }
 
-  _logExecOutput(log, result, verbosity === 'high' ? 'debug' : 'trace');
+  _logExecOutput(log, result, verbosity === 'high' ? 'info' : 'trace');
```

Comment:
Are you sure we need to print adb devices output in absolutely all cases, @d4vidi? We all know that there is no other way to get into Telnet error other than entering that unfortunate `if` branch. So why pollute logs for other people?

Reply:
I can't see how 2-3 lines can pollute anything, but nevertheless after adding detailed logging this is indeed pointless...